src/main/java/org/apache/accumulo/examples/client/SequentialBatchWriter.java

```diff
@@ -52,14 +49,42 @@ public static Value createValue(long rowId) {
     return new Value(value);
   }
 
+  static class Opts extends ClientOpts {
```

Comment:
As an aside (not for this issue... but perhaps in future), I think it'd be fine if the examples stopped using JCommander and just had options hard-coded. As examples, they are intended to be user-editable, so the simpler they are, the better. You can't get much simpler than hard-coded literals at the beginning of "main".